eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/HttpCommand.java

```diff
@@ -57,9 +57,27 @@ public class HttpCommand implements ProtocolTransportObject {
 
     public String httpVersion;
 
-    public Header header;
+    private transient Header header;
 
-    public Body body;
+    private transient Body body;
+    
+    // getter and setter for header
+    public Header getHeader() {
+        return header;
+    }
+
+    public void setHeader(Header header) {
+        this.header = header;
+    }
+
+    // getter and setter for body
+    public Body getBody() {
+        return body;
+    }
+
+    public void setBody(Body body) {
+        this.body = body;
+    }
```

Comment:
This class has @Data annotation by lombok, do not need to add additional getter/setter